src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base.js

```diff
@@ -117,6 +117,12 @@ export default class EzConfigBase {
         };
     }
 
+    addExtraButtons(extraButtons = []) {
```

Comment:
`extraButtons` is an object so `extraButtons = {}`

Reply:
@dew326 done: https://github.com/ezsystems/ezplatform-admin-ui/pull/943/commits/98688a73c2c6a9db737e5d24d77d60882209da70

Reply:
Done:
- https://github.com/ezsystems/ezplatform-admin-ui/pull/943/commits/ff8f2705acc2c61dbee68be2574497309c49123e
- https://github.com/ezsystems/ezplatform-admin-ui/pull/943/commits/2613486e7dc7cde8d0972a88be97a21e0a7a4a83